kratos/utilities/nurbs_curve_tessellation.h

```diff
@@ -108,14 +108,14 @@ class NurbsCurveTessellation
         const int PolynomialDegree,
         const NurbsInterval DomainInterval,
         const std::vector<NurbsInterval>& rKnotSpanIntervals,
-        const double Tolerance
+        const double Tolerance,
+        bool ToSurfaceParameter = false
```

Comment:
missing doc, would be nice to know what this actually means

Reply:
yo, done. thanks for that comment!